packages/components/music/TrackCard.tsx

```diff
@@ -81,9 +83,12 @@ export const TrackCard: React.FC<{
           onHoverOut={() => setIsHovered(false)}
           style={imgBoxStyle}
           onPress={() => {
-            navigation.navigate("FeedPostView", {
-              id: getNetworkObjectId(selectedNetworkId, post.identifier),
-            });
+            navigation.navigate(
+              appMode === "mini" ? "MiniFeedDetails" : "FeedPostView",
```

Comment:
the screen should change depending on mode instead of navigating to different screens depending on mode